sdk/src/wallet/mod.rs

```diff
@@ -87,10 +88,10 @@ use crate::{
 #[derive(Debug, Default, Clone, Serialize, Deserialize, Eq, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct FilterOptions {
-    /// Filter all outputs where the booked milestone index is below the specified timestamp
-    pub lower_bound_booked_timestamp: Option<u32>,
-    /// Filter all outputs where the booked milestone index is above the specified timestamp
-    pub upper_bound_booked_timestamp: Option<u32>,
+    /// Filter all outputs where the included slot is below the specified slot.
+    pub included_below_slot: Option<SlotIndex>,
+    /// Filter all outputs where the included slot is above the specified slot.
+    pub included_above_slot: Option<SlotIndex>,
```

Comment:
Maybe it's just me, but to with these names it's not clear if it would return outputs that match the condition or output that don't match it

Reply:
Maybe it's enough if the comments say what exactly happens
Currently seems to do the opposite of the Rust iter filter() method which does this
````
Given an element the closure must return true or false. The returned iterator will yield only the elements for which the closure returns true.
```

Reply:
Actually I think I messed it up and did the opposite

Reply:
I reversed them, do you still want me to update a comment or something?

Reply:
Hm I think you reversed the "meaning" compared to what it was before now
Could still make sense though, but yes please update the comment so it's clear if an output that matches the condition will be returned or not